shadow-dom/focus/resources/shadow-utils.js

```diff
@@ -60,8 +60,13 @@ function removeTabIndex(elements) {
   }
 }
 
+function resetFocusOnRoot(root) {
```

Comment:
Can we just have one function, instead of two?

Reply:
`resetFocus()` is already used by the sequential focus navigation tests so we still need it (unless we change it too I guess?), and we need to give the root here because we don't know the tree structure?

Reply:
I see, I think I misread things. I'm OK with merging this as-is. However it could be clearer to just have `function resetFocus(root = document) { /* contents of current resetFocusOnRoot */ }` and update all tests to use that.

Reply:
Oh yeah sorry the non-document one is not actually used in this WPT, oops (but will be used in other ones so I think it's OK to have the general version here). Also changed it to your suggested version. Thanks!